backend/utils/room_utils.py

```diff
@@ -58,7 +58,7 @@ async def get_room(org_id: str, room_id: str) -> dict:
 
     if response and "status_code" not in response:
         return response
-    return {}
+    return None
```

Comment:
Have u faced an issue with the function returning an empty dict if there is no room found?

Reply:
yes I did, an empty dict is not a none value, so when a room is not returned the error is not caught and the code fails

Reply:
I see. But empty dicts evaluate to False.
Therefore, `if not room` could have worked.
@mykie88

Reply:
yes it would have, but this seems like a matter of preference, and quite inconsequential. How does it affect anything?

Reply:
{} vs None battle loading...

Reply:
Lmao... let me just be the bigger person